src/components/Shared/AnnotationMenu/AnnotationMenuComponent.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import {IconName, MenuItem} from "@blueprintjs/core";
+import {CARTA} from "carta-protobuf";
+import {observer} from "mobx-react";
+
+import {CustomIcon, CustomIconName} from "icons/CustomIcons";
+import {AppStore} from "stores";
+import {RegionMode, RegionStore} from "stores/Frame";
+
+@observer
+export class AnnotationMenuComponent extends React.Component {
+    handleRegionTypeClicked = (type: CARTA.RegionType) => {
```

Comment:
Perhaps we can set `handleRegionTypeClicked` as a prop and reuse `handleRegionTypeClicked` in `ToolbarComponent` and `ToolbarMenuComponent`. These two functions are actually slightly different. I think we can try to use functional components to make the code simpler.